sdk/core/azure-core-tracing-opentelemetry/azure/core/tracing/ext/opentelemetry_span/__init__.py

```diff
@@ -277,21 +283,23 @@ def get_trace_parent(self) -> str:
 
     @classmethod
     def link(cls, traceparent: str, attributes: Attributes = None) -> None:
-        """
-        Links the context to the current tracer.
+        """Links the context to the current tracer.
 
         :param traceparent: A complete traceparent
         :type traceparent: str
+        :param attributes: Attributes to be added to the link
+        :type attributes: dict or None
         """
         cls.link_from_headers({"traceparent": traceparent}, attributes)
 
     @classmethod
-    def link_from_headers(cls, headers: Dict[str, str], attributes: Attributes = None):
-        """
-        Given a dictionary, extracts the context and links the context to the current tracer.
+    def link_from_headers(cls, headers: Dict[str, str], attributes: Attributes = None) -> None:
+        """Given a dictionary, extracts the context and links the context to the current tracer.
 
         :param headers: A key value pair dictionary
         :type headers: dict
+        :param attributes: Attributes to be added to the link
+        :type attributes: dict or None
```

Comment:
Is type Attributes (line 289) just a dict?

Reply:
An Optional Dict 😃 

Definition at the top bakes in the Optional: `Attributes = Optional[Dict[str, AttributeValue]]`.